src/services/date-utilities.ts

```diff
@@ -368,8 +368,8 @@ export default class DateUtilities {
    * @example (18, 2023-02-03) -> "2024-08-03"
    */
   static addMonthsToDate (month: number, date: string): string {
-    if (!date) {
-      date = '2023-01-01'
+    if (!month || !date) {
+      return null
```

Comment:
This is to address [this comment](https://github.com/bcgov/business-filings-ui/pull/577#discussion_r1389645250) on Karim's previous PR